public/app/features/alerting/unified/utils/cloud-alertmanager-notifier-types.ts

```diff
@@ -377,6 +377,12 @@ export const cloudNotifierTypes: Array<NotifierDTO<CloudNotifierType>> = [
       option('message', 'Message', 'Message template', {
         placeholder: '{{ template "webex.default.message" .}}',
       }),
+      option('disable_web_page_preview', 'Disable web page preview', 'Disable web page preview in telegram messages', {
+        element: 'checkbox',
+      }),
+      option('protect_content', 'Protect content', 'Disable forward and save messages on telegram', {
+        element: 'checkbox',
+      }),
```

Comment:
My understanding that these are options of the cloud Alertmanager, that uses the notifiers of Prometheus Alertmanager. Afaiu, you did not change it, did you?
UI for Grafana-managed alerts is automatically built upon changes you made in the .go file.

Reply:
Ditto @yuri-tceretian, this is for Prometheus' Alertmanager's notifiers only